src/components/buttons/index.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react'
+import Link from 'next/link'
+import noop from 'utils/noop'
+
+type ButtonProps = {
+  url: string
+  label: string
+  className?: string
+  quiet?: boolean
+  onClick?: () => void
+}
+
+export const PrimaryButton: React.FC<ButtonProps> = ({
+  url,
+  className,
+  label,
+  onClick = noop,
+}) => {
+  return (
+    <Link href={url}>
```

Comment:
suggestion: if we are going to really dig in to standardize a single common button is that we should be focused on a11y and semantics. There are some distinctive differences between button buttons and link buttons that should be accounted for.

Any clickable element that opens a new page is best represented by a link/anchor. This might also handle an `onClick` in our app, but that is reserved for analytics and not interaction.

An element that performs an action when clicked should be represented by the `button` element or have the appropriate aria role assigned to it.

Marcy Sutton has a great article with more in-depth details about this: https://marcysutton.com/links-vs-buttons-in-modern-web-applications

This is a well-considered article with React code that demonstrates a very solid solution/abstraction for this issue: https://kittygiraudel.com/2020/01/17/accessible-links-and-buttons-with-react/